lib/src/screens/explore/user_screen.dart

```diff
@@ -218,14 +291,49 @@ class _UserScreenState extends State<UserScreen> {
                             ],
                           )
                         ]),
-                      if (_data!.about != null)
+                      if (_data!.about != null || _aboutTextController != null)
                         Padding(
                           padding: const EdgeInsets.only(top: 12),
-                          child: Markdown(
-                            _data!.about!,
-                            getNameHost(context, _data!.username),
-                          ),
-                        )
+                          child: _aboutTextController == null
+                              ? Markdown(
+                                _data!.about!,
+                                getNameHost(context, _data!.username),
+                              )
+                              : TextEditor(
+                                _aboutTextController!,
+                                label: "About",
+                              )
```

Comment:
Looks good. The only thing needed is for the "About" text editor to have `isMarkdown` set to `true`. That way, multi-line input is enabled.